openlibrary/plugins/upstream/borrow.py

```diff
@@ -842,6 +843,16 @@ def get_ia_auth_dict(user, item_id, user_specified_loan_key, access_token):
         'token': make_ia_token(item_id, BOOKREADER_AUTH_SECONDS)
     }
 
+def make_access_key():
+    try:
+        access_key = config.ia_access_secret
+        if not isinstance(access_key, (bytes, bytearray)):
+            access_key = access_key.encode('utf-8')
+        return access_key
+    except AttributeError:
+        raise RuntimeError(
+            "config value config.ia_access_secret is not present -- check your config"
+        )
```

Comment:
Minimize the scope of the exception...
```suggestion
    except AttributeError:
        raise RuntimeError(
            "config value config.ia_access_secret is not present -- check your config"
        )
    if not isinstance(access_key, (bytes, bytearray)):
        access_key = access_key.encode('utf-8')
    return access_key
```